spec_parser/utils.py

```diff
@@ -658,5 +681,6 @@ def _gen_rdf(self, g: rdflib.Graph):
 
         # add entries
         for _entry, _desc in self.entries.items():
-            g.add((self._gen_uri(_entry), RDF.type, OWL.NamedIndividual))
-            g.add((self._gen_uri(_entry), RDF.type, cur))
+            uri = cur + "/" + _entry
+            g.add((uri, RDF.type, OWL.NamedIndividual))
+            g.add((uri, RDF.type, cur))
```

Comment:
Why did you change this?

Reply:
That's the fix for the first issue - `_gen_uri` generates an inccorect URIRef for the vocab elements that's missing the vocab type prefix.